net/htpdate/Makefile

```diff
@@ -50,8 +51,8 @@ endef
 define Package/htpdate/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/htpdate $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/default/
-	$(INSTALL_CONF) ./files/htpdate.default $(1)/etc/default/htpdate
+	$(INSTALL_DIR) $(1)/etc/config/
+	$(INSTALL_CONF) ./files/htpdate.conf $(1)/etc/config/htpdate
```

Comment:
you forgot to update ```define Package/htpdate/conffiles```

Reply:
You're right, please check updated commit.